src/app.test.js

```diff
@@ -1,10 +1,27 @@
-import { addTask, removeTask } from './modules/store';
+import {
+  addTask, removeTask, editTask, completed, clearCompleted,
+} from './modules/store';
 
 describe('Add task', () => {
   localStorage.removeItem('tasks');
   test('Add task "sleep"', () => {
     expect(addTask('sleep')).toEqual([{ completed: false, description: 'sleep', index: 1 }]);
   });
+  test('Add task "cook"', () => {
+    expect(addTask('cook')).toEqual([{ completed: false, description: 'sleep', index: 1 }, { completed: false, description: 'cook', index: 2 }]);
+  });
+  test('Edit task "sleep"', () => {
+    editTask(1, 'sleeping');
+    expect(JSON.parse(localStorage.getItem('tasks'))).toEqual([{ completed: false, description: 'sleeping', index: 1 }, { completed: false, description: 'cook', index: 2 }]);
+  });
+  test('Completed task "sleeping"', () => {
```

Comment:
- [ ] Great job adding the pure fuctions test. As per project requirements, your are to use [mock](  https://robertmarshall.dev/blog/how-to-mock-local-storage-in-jest-tests/ ) for functions that use `localstorage`, `DOM` and `external APIs`. But I cannot find the use of mocks anywhere.  Kindly add this.

Reply:
hello @t-yanick thank you for your review, in the file __setups__/localstorage.js you'll see the necessary imports

Reply:
Thanks for the reply. Kindly submit for re-review and the next reviewer will approve the PR.

Reply:
Thank you